miasm2/arch/mips32/jit.py

```diff
@@ -69,12 +69,13 @@ def gen_finalize(self, block):
         Generate the C code for the final block instruction
         """
 
-        lbl = self.get_block_post_label(block)
-        out = (self.CODE_RETURN_NO_EXCEPTION % (self.label_to_jitlabel(lbl),
+        loc_key = self.get_block_post_label(block)
+        offset = self.ir_arch.symbol_pool.loc_key_to_offset(loc_key)
+        out = (self.CODE_RETURN_NO_EXCEPTION % (self.label_to_jitlabel(loc_key),
```

Comment:
`label_to_jitlabel` no more use a `label`, but a `loc_key` instead
Maybe it can be renamed

Reply:
done